### DISCUSSION TITLE
Предупреждение гугла: [Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event.

### DISCUSSION THREAD

**Discussion by Vladislav-Chistyakov** [yandex-maps-unofficial/vue-yandex-maps#349] [Q&A (2.x)] [answered]
При разработке лейндинга на SSR, подключил новую версию vue-yandex-map. Стараюсь оптимизировать Seo, и при загрузке карты вылетает предупреждение в консоли: [Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See [site](https://www.chromestatus.com/feature/5745543795965952).

Вылетает при 'touchstart' и 'touchmove'

![image](https://github.com/user-attachments/assets/0c778ad4-e9f7-4bc1-ab3f-9bf1bb5d202e)
![image](https://github.com/user-attachments/assets/f0dbdd99-39f5-464b-963a-0e711dbee2ae)

Создано ли решение этой проблемы у вас или вы ещё с таким не сталкивались?

Нашёл решения, их достаточно много. Присмотритесь пожалуйста
https://github.com/angular/components/issues/4221#issuecomment-299872704

P.S. daniluk4000 Проходил у тебя собес в Азбуку Весной, наконец то добрался до твоих разработок карт на vue!  :)

**Comment by daniluk4000** [comment, marked as answer]
Приветики! Это добавляет Яндекс, так что все благодарности им в форму обратной связи))

**Reply by Vladislav-Chistyakov** [reply]
Эххх, да как так...))
Спасибо за быстрый ответ))
Отличный проект!
